hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/key/TestOMKeyResponse.java

```diff
@@ -76,7 +76,6 @@ public void setup() throws Exception {
     random = new Random();
   }
 
```

Comment:
I still confused why it is complaining.
I see code of getOpenKey:
 ```
 @Override
  public String getOpenKey(String volume, String bucket,
                           String key, long id) {
    String openKey = OM_KEY_PREFIX + volume + OM_KEY_PREFIX + bucket +
        OM_KEY_PREFIX + key + OM_KEY_PREFIX + id;
    return openKey;
  }
```
  
Looks like it will not be null. Not sure whats going on.

Reply:
The findbugs complaints were mostly "redundant null check for value known to not be null", so removing the annotation made the warning go away.